README.md

```diff
@@ -41,7 +41,7 @@ Are you a talented programmer looking to contribute some code? We'd love the hel
 * Please follow the above guidelines for your pull request(s) accepted.
 * For help setting up the project, keep reading!
 
-Love the project? Feel free to [donate] to help continue development! Flow projects are open-source and powered by community members, like yourself. Without you, we wouldn't be here today!
+Love the project? Feel free to donate to help continue development! Flow projects are open-source and powered by community members, like yourself. Without you, we wouldn't be here today!
```

Comment:
You can remove this line altogether. The Glowstone project doesn't accept donations at the moment.